### PR TITLE
Command parser is now checked for formatting

### PR DESCRIPTION
**Pull request by liealex**

### CODE REVIEW

File: .clangformatignore [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apps/uvc-app/include
 apps/uvc-app/lib
 bindings/python/pybind11
-dependencies/
+dependencies/third_party
```
Comment (dNechita) [inline .clangformatignore:4]: why third_party folder? @liealex

Comment (dNechita) [inline .clangformatignore:4]: the actual name of the folder is "third-party" with  a dash instead of underscore

Comment (dNechita) [reply .clangformatignore:4]: that's why CI is failing

Comment (liealex) [reply .clangformatignore]: Solved

File: .clangformatignore [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+dependencies/third-party
 drivers/
 examples/data_collect/third_party
 examples/data_collect/adi
 examples/tof-viewer/external
 examples/tof-viewer/include/stb_image.h
 examples/tof-viewer/include/linmath.h
 sdk/common/adi/ccb
-sdk/src/cameras/itof-camera/tofi
+sdk/src/cameras/itof-camera/tofi
```
Comment (SeptimiuVana) [inline .clangformatignore:12]: Missing newline at the end of file.

**Review by SeptimiuVana — APPROVED** [review]

**Pull request merged**